packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount()
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with owner as all', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount('all')
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with no_rewards option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          no_rewards: true
+        }
+        const count = await client.account.historyCount('mine', options)
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with token option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          token: 'DBTC'
+        }
+        const count = await client.account.historyCount('mine', options)
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistory with txtype option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          txtype: TxType.MINT_TOKEN
+        }
```

Comment:
this is same as well, you can make use of `tytype` to expect the diff exact count

Reply:
Sure. Will write and additional test case for that.

Reply:
this part is actually same principle with https://github.com/DeFiCh/jellyfish/pull/265/files#r638661038 comment
test with diff txtype and assert the exact number